app/assets/stylesheets/content/_forms.sass

```diff
@@ -552,3 +552,11 @@ fieldset.form--fieldset
 
   li, div
     padding: 0
+
+// change password form
+form.change_password
+  max-width: 31em
+  min-height: 31em
```

Comment:
... why?

This is a super specific form currently, which should behave like any other form in the current application. It neither needs a maximum width nor a maximum height.